api-reference/10 UI Widgets/GridBase/1 Configuration/onKeyDown.md

```diff
@@ -18,7 +18,7 @@ The widget's instance.
 #include common-ref-elementparam with { element: "widget" }
 
 ##### field(e.event): event
-#include common-ref-eventparam
+The event that caused the function to execute. It is a [dxEvent](/api-reference/50%20Common/Object%20Structures/dxEvent '/Documentation/ApiReference/Common/Object_Structures/dxEvent/') or a <a href="http://api.jquery.com/category/events/event-object/" target="_blank">jQuery.Event</a> when you use jQuery. The underlying native event is <a href="https://developer.mozilla.org/en-US/docs/Web/API/Document/keydown_event" target="_blank">keydown</a>.
```

Comment:
This paragraph makes no sense. The sentences don't tie into each other. Please rewrite.

Reply:
Well, it is a commonly used description for the function parameter. We use so often that we moved it to a separate file -  #include common-ref-eventparam. But here I need to extend this description and mention that our event uses the native Javascript event - [onkeydown](https://developer.mozilla.org/en-US/docs/Web/API/Document/keydown_event) and extends it by adding some functionality. That's why I called it "an underlying event"
![image](https://user-images.githubusercontent.com/33040934/73159998-e702aa00-40f8-11ea-9232-b4987aa09288.png)

Reply:
ok, Julian said I can leave it as is too.